### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by thuBingo** [ericsun99/Shufflenet-v2-Pytorch-#13]
I can not unzip the file by:
tar -xvf shufflenetv2_x1_69.402_88.374.pth.tar
How can I use this pre-trained parameter?
Thank you.
